docs/user-guide/vsphere-machinesets.md

```diff
@@ -0,0 +1,951 @@
+```
+
+**Status columns:**
+
+- **DESIRED** - What you configured in `spec.replicas`
+- **CURRENT** - How many Machines the Machine API created
+- **READY** - How many Machines have successfully joined the cluster
+- **AVAILABLE** - How many Machines are ready and available
+
+All columns should match when the MachineSet is healthy.
+
+### 8.2 Verify Machine creation and lifecycle
+
+Watch the Machine object as it progresses through provisioning phases:
+
+```bash
+oc get machines -n openshift-machine-api -w
+```
+
+**Expected progression:**
+
+1. **Provisioning** - VM is being cloned from template
+2. **Provisioned** - VM created and powered on
+3. **Running** - Node has joined the cluster
+4. Machine gains a `NodeRef` (confirms it successfully joined)
+
+**Inspect the Machine details:**
+
+```bash
+oc describe machine -n openshift-machine-api <machine-name>
+```
+
+**Look for successful status:**
+
+```
+Status:
+  Phase: Running
+  Node Ref:
+    Kind: Node
+    Name: my-test-cluster-d7-jzsct-worker-xxxxx
+    UID: ...
+```
+
+The presence of `Node Ref` confirms the Machine successfully registered a node in the cluster.
+
+### 8.3 Confirm CSRs are approved
+
+List certificate signing requests:
+
+```bash
+oc get csr
+```
+
+**Expected behavior:**
+
+On most OpenShift 4.15+ clusters, worker CSRs auto-approve by default (unless cluster admins disabled it via custom RBAC). If you see `Pending` CSRs for more than 2 minutes, approve them manually:
+
+```bash
+# Approve all pending CSRs
+oc get csr -o name | xargs oc adm certificate approve
+```
+
+**Expected output (after approval):**
+
+```
+NAME        AGE   SIGNERNAME                                    REQUESTOR                       CONDITION
+csr-abcde   40s   kubernetes.io/kube-apiserver-client-kubelet   system:serviceaccount:...       Approved,Issued
+csr-fghij   20s   kubernetes.io/kubelet-serving                 system:node:worker-xxxxx        Approved,Issued
+```
+
+### 8.4 Verify the new node joins the cluster
+
+Check that the node appears and reaches `Ready` status:
+
+```bash
+oc get nodes -o wide
+```
+
+**Expected output:**
+
+```
+NAME                                        STATUS   ROLES    AGE     INTERNAL-IP    EXTERNAL-IP   OS-IMAGE
+my-test-cluster-d7-jzsct-master-0        Ready    master   1d      10.184.6.171   <none>        Red Hat Enterprise Linux CoreOS ...
+my-test-cluster-d7-jzsct-master-1        Ready    master   1d      10.184.6.172   <none>        Red Hat Enterprise Linux CoreOS ...
+my-test-cluster-d7-jzsct-master-2        Ready    master   1d      10.184.6.173   <none>        Red Hat Enterprise Linux CoreOS ...
+my-test-cluster-d7-jzsct-worker-xxxxx    Ready    worker   2m      10.184.6.174   <none>        Red Hat Enterprise Linux CoreOS ...
+```
+
+**Success criteria:**
+
+- ✅ New worker appears in the list
+- ✅ Status is `Ready`
+- ✅ Age is recent (seconds or minutes)
+- ✅ IP address is from the correct network range
+
+---
+
+## 9. Test MachineSet scaling
```

Comment:
This is all documented in OCP docs right?

I think we should keep this specific to what is different in assisted from installing a regular IPI cluster with vsphere integration